crontab.tools

```diff
@@ -4,6 +4,7 @@ PYTHONPATH=$HOME/.local/lib/python2.7/site-packages:$PYTHONPATH
 
 # INSTRUCTIONS
 # MIN HOUR DAYMONTH MONTH DAYWEEK
+@hourly jsub -once -mem 2G -N rusty /data/project/dbreps/src/database-reports/target/release/dbreps2
```

Comment:
how come this can use `@hourly` but the rest require that the minute/hour/daymonth/month/dayweek all need to be specified?

Reply:
Because...those don't need to run every hour? The idea in the new system is that *everything* runs every hour, and each report checks whether its out of date by parsing the "data as of" line, and if it's within the window, it gets skipped.

See also https://en.wikipedia.org/wiki/Cron#Nonstandard_predefined_scheduling_definitions